application/change.mjs

```diff
@@ -29,6 +30,7 @@ export default async function change (institutions) {
     const institution = institutions.shift()
     const institutionId = getInstitutionId(institution)
     const institutionName = getInstitutionName(institution)
+    const targetInstitutionMetaData = createMetaData(institution)
```

Comment:
Hmm, here we have the same problem when updating the metadata I think. Maybe we should refactor this `createMetadata` function to be reused across createOrganisation.mjs and change.mjs : 

```
createMetaData(organizationMetaData, {institutionId}))
```

the body of the function could be: 

````js
return {
... organisationMetadata ?? {},
... targetInstitutionMetadata
}